examples/ci-tests/i2c-master-rx/README.md

```diff
@@ -1,21 +1,64 @@
-# Hardware CI Tests
+# I2C Master RX Test
 
-## Run Python Test
+## Test Description
 
-`test.py` provides basic test that sends message from slave to master, and waits for master to send message containing the slave message. This is essentially a Rx/Tx test, but we are primarily focusing on the reception of master here. The message sent from master will be printed over a 12 second period. If message sent from master is the exact same from message sent from slave, the test will pass.
+`test.py` provides the following tests:
+
+### I2CMasterRxTest
+
+This receives message from the device-under-test as I2C Master from the RPi as I2C Slave.
+
+  1. The device-under-test configures itself as I2C master with address 0x40 and awaits button press.
+  1. The test harness presses a button (GPIO toggle) on device-under-test to indicate the device-under-test to prepare interaction and receive message as master.
+  1. The test harness issues an I2C write transaction of 16 bytes and awaits master to send back read data.
+  1. The test harness validates that the data received from device-under-test and sent back to test harness matches the expected string sent `"Hello I'm Slave"`.
+
+## Example Output
+
+      0.000194 INFO -- Initiating I2C Master Rx Test...
+      1.017061 INFO -- Setting up for nrf52840dk I2C Master Rx test...
+      1.017617 INFO -- Sending Messages As Slave... 
```

Comment:
This is maybe a confusing message for the test application to print.

Perhaps instead of `Sending Messages As Slave...` it should be `Test Harness Preparing to Respond to Read Transactions as Slave at Address 0x41`

A little verbose, but also very clear.